src/olympia/addons/migrations/0020_backfill_hashed_guids.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.15 on 2020-08-05 13:57
+
+from django.db import migrations
+from django.db.models import Func, F
+
+
+def backfill_hashed_guids(apps, schema_editor):
+    AddonGUID = apps.get_model("addons", "AddonGUID")
+    AddonGUID.objects.filter(hashed_guid=None).update(
+        hashed_guid=Func(F("guid"), 256, function="SHA2")
```

Comment:
call me suspicious, but are we 100% sure this is producing the same hash as `hashlib.sha256(self.guid.encode()).hexdigest()`?

Reply:
Yeah, I checked manually a few GUIDs against the FF implementation, an online too to compute SHA256 hashes, the MySQL function, and the python implementation.

Reply:
I'm not worried about any differences since guids are supposed to be ascii so there shouldn't be any implementation surprises, but I would rather do this in small chunks using a dedicated `adhoc` task (computing the hashes in python to leave the database some time to breathe). Computing and storing millions of hashes in one query is probably not very nice on the db...

Could you check with ops ? If they say it's fine I don't mind, but otherwise, let's go the task route.

Reply:
@bqbn what do you think? This query should perform an `UPDATE` on `addons_reusedguid` and will update all rows.

Reply:
If there are millions of hashes to be processed by the db, then definitely take the task route!

I don't think no one wants to sit in the push window and wait too long for this to complete. Besides, db migrations on the Addon table often cause metadata locks in the replicas.

Thanks for catching this @diox. :)

Reply:
@bqbn That would be on the `addons_reusedguid` table. How many rows do we have on prod?

Reply:
1.9 millions.

Reply:
Okay, that's a lot. I will update this PR to remove the backfill migration, and I'll work on a new backfill solution when I am back.

Reply:
done.